cou/utils/__init__.py

```diff
@@ -14,8 +14,7 @@
 
 """Utilities for charmed-openstack-upgrader."""
 
-
 import os
 from pathlib import Path
 
-COU_DATA = Path(os.getenv("COU_DATA", "."))
+COU_DATA = Path(f"{os.getenv('HOME')}/.local/share/cou")
```

Comment:
Putting an external discussion here for future references: `$HOME` variable has a different value inside non-classic snaps ([ref](https://snapcraft.io/docs/environment-variables#heading--home)), so logs would not be saved to the expected system path. Perhaps we should consider save logs to a directory inside $SNAP_COMMON or $SNAP_DATA instead.

Reply:
The problem of using [SNAP_COMMON or SNAP_DATA](https://snapcraft.io/docs/environment-variables#heading--snap-common)  it's writable just for root and that is not our case. If we were running a daemon, than this could be done